code/modules/projectiles/projectile/beams.dm

```diff
@@ -290,26 +291,38 @@ var/list/beam_master = list()
 		kill_count--
 		bump_original_check()
 
+		var/AX = (starting.x - src.x)*32

```

Comment:
it's the same algo as update_pixel() but it's usage here is quite different, since the pixel_offset isn't applied to the atom.